openedx_webhooks/github_views.py

```diff
@@ -48,14 +47,24 @@ def pull_request():
 
     pr = event["pull_request"]
     repo = pr["base"]["repo"]["full_name"].decode('utf-8')
-    if event["action"] == "opened":
-        return pr_opened(pr)
-    if event["action"] == "closed":
-        return pr_closed(pr)
     if event["action"] == "labeled":
         return "Ignoring labeling events from github", 200
     if event["action"] == "synchronize":
         return "Ignoring synchronize events from github", 200
+    if event["action"] == "opened":
+        result = pull_request_opened.delay(pr, wsgi_environ=minimal_wsgi_environ())
+        status_url = url_for("tasks.status", task_id=result.id, _external=True)
+        resp = jsonify({"message": "queued", "status_url": status_url})
+        resp.status_code = 202
+        resp.headers["Location"] = status_url
+        return resp
+    if event["action"] == "closed":
+        result = pull_request_closed.delay(pr)
```

Comment:
This is the only line that seems to differ from the code under `if event["action"] == "opened"` so maybe,

```python
if event["action"] == "opened":
    result = pull_request_opened.delay(pr, wsgi_environ=minimal_wsgi_environ())
elif event["action"] == "closed":
    result = pull_request_closed.delay(pr)
else:
    # some error message? unknown action?
status_url = url_for("tasks.status", task_id=result.id, _external=True)
....
return resp
```